tests/unit/src/kakarot/instructions/test_environmental_information.cairo

```diff
@@ -140,3 +140,39 @@ func test__exec_extcodecopy__should_handle_address_with_no_code{
 
     return ();
 }
+
+@view
+func test__returndatacopy{
+    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
+}() {
+    // Given
+    alloc_locals;
+
+    let (bytecode) = alloc();
+    let return_data: felt* = alloc();
+    let return_data_len: felt = 32;
+    // filling at return_data + 1 because first first felt is return_data offset
+    TestHelpers._fill_bytecode_with_values(return_data + 1, 32, 0xFF);
+    let child_ctx: model.ExecutionContext* = TestHelpers.init_context_with_return_data(
+        0, bytecode, return_data_len, return_data
+    );
+
+    let stack: model.Stack* = Stack.init();
+    let stack: model.Stack* = Stack.push(stack, Uint256(32, 0));
+    let stack: model.Stack* = Stack.push(stack, Uint256(0, 0));
+    let stack: model.Stack* = Stack.push(stack, Uint256(0, 0));
+    let ctx: model.ExecutionContext* = TestHelpers.init_context_with_stack_and_sub_ctx(
+        0, bytecode, stack, child_ctx
+    );
+
+    // When
+    let result: model.ExecutionContext* = EnvironmentalInformation.exec_returndatacopy(ctx);
+
+    // Then
+    let (memory, data) = Memory.load(result.memory, 0);
+    assert_uint256_eq(
+        data, Uint256(0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF, 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF)
+    );
+    assert result.gas_used = 3;
```

Comment:
Can you add another test with an offset? For example the same second case than in playground

<img width="588" alt="image" src="https://user-images.githubusercontent.com/1172099/206477978-566e6d0b-c692-48ca-8da3-517dd54c0d65.png">